addons/mail/static/src/views/web/activity/activity_renderer.xml

```diff
@@ -65,9 +65,11 @@
 <t t-name="mail.ActivityViewRow" owl="1">
     <tr class="o_data_row h-100" t-attf-class="{{ activeFilter.resIds.includes(resId) ? 'o_activity_filter_' + activeFilter.progressValue.active : '' }}">
         <t t-set="record" t-value="getRecord(resId)"/>
-        <ActivityRecord archInfo="props.archInfo" record="record" openRecord="props.openRecord" />
-        <t t-foreach="activeColumns" t-as="type" t-key="type[0]">
-            <t t-call="mail.ActivityViewCell"/>
+        <t t-if="record">
```

Comment:
Does it make sense to have ActivityViewRow for not existing record? Shouldn't the condition be in parents instead?

Reply:
I just put it here to stop the crash while keeping the feature working as expected. `ActivityRecord` and `ActivityViewCell` require `record` being set, so that's why I put it here